commands/import.go

```diff
@@ -0,0 +1,251 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/spf13/cobra"
+)
+
+const fakeUserID = "nouser"
+
+var ImportCmd = &cobra.Command{
+	Use:   "import",
+	Short: "Management of imports",
+}
+
+var ImportUploadCmd = &cobra.Command{
+	Use:     "upload [filepath]",
+	Short:   "Upload import files",
+	Example: " import upload import_file.zip",
+	Args:    cobra.ExactArgs(1),
+	RunE:    withClient(importUploadCmdF),
+}
+
+var ImportListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List import files",
+}
+
+var ImportListAvailableCmd = &cobra.Command{
+	Use:     "available",
+	Short:   "List available import files",
+	Example: " import list available",
+	Args:    cobra.NoArgs,
+	RunE:    withClient(importListAvailableCmdF),
+}
+
+var ImportListIncompleteCmd = &cobra.Command{
+	Use:     "incomplete",
+	Short:   "List incomplete import files uploads",
+	Example: " import list incomplete",
+	Args:    cobra.NoArgs,
+	RunE:    withClient(importListIncompleteCmdF),
+}
+
+var ImportListJobsCmd = &cobra.Command{
+	Use:     "jobs [importJobID]",
+	Example: " import list jobs",
+	Short:   "List import jobs",
+	Args:    cobra.MaximumNArgs(1),
```

Comment:
nit: What do you think of defining a custom function to check number of args and if it doesn't match the expectation we can return an error like "import job ID is required".